cscore/src/main/java/edu/wpi/first/cscore/VideoException.java

```diff
@@ -16,9 +16,4 @@ public class VideoException extends RuntimeException {
   public VideoException(String msg) {
     super(msg);
   }
-
-  @Override
-  public String toString() {
-    return "VideoException [" + super.toString() + "]";
```

Comment:
💭 This didn't seem particularly useful — anything which shows the exception message should also show the class name, and we don't do this anywhere else.